pkg/resource/asset.go

```diff
@@ -754,6 +754,11 @@ func (r *directoryArchiveReader) Next() (string, *Blob, error) {
 	}
 	name = filepath.Clean(name)
 
+	// Replace Windows-style separators with Linux ones (if we are on Windows)
+	if os.PathSeparator == '\\' {
```

Comment:
Couldn't we just do this all the time?  This is a no-op on platforms where the separator is already a forward slash (https://golang.org/src/path/filepath/path.go?s=4426:4458#L155)

Reply:
Ha! I did not check the source code. Based on the comment, I was guessing that they replace regardless, and then some backslashes in file names would be replaced.